### PR TITLE
feat: add event rows to Show Page Calendar tab

### LINKED ISSUE #4287: Show Page/Activities Panel/Calendar Tab - Add event rows
## Context

For this issue:
- In the Show Page/Activities Panel/Calendar Tab, display Calendar events as rows grouped by days and months (1 card per month).
- Past days are displayed with a white background, present and future days are displayed with a grey background (see Figma for exact colors).
- Past events are displayed with a normal font weight, current and future events are displayed with a bold font weight (see Figma for exact font weights).
- Use mock data for now.
- For now do not implement:
  - Month headers
  - Participants avatars group
  - Next event red line

## Figma

Image

https://www.figma.com/file/xt8O9mFeLl46C5InWwoMrN/Twenty?type=design&node-id=23916%3A43344&mode=design&t=f5B7MSvEHciAWa3O-1

### PR DESCRIPTION
**Pull request by thaisguigon**
Closes #4287

### CODE REVIEW

File: packages/twenty-front/src/modules/activities/calendar/types/CalendarEvent.ts
Comment (bosiraphael) [inline packages/twenty-front/src/modules/activities/calendar/types/CalendarEvent.ts:1]: We changed the type: startDate and endDate are now startsAt and endsAt, status has been replaced by a boolean isCanceled and visibility is 'METADATA' | 'SHARE_EVERYTHING'

**Review by bosiraphael — APPROVED** [review]
LGTM, I left a comment about the changes in the data model

File: packages/twenty-front/src/modules/activities/calendar/types/CalendarEvent.ts
Comment (thaisguigon) [reply packages/twenty-front/src/modules/activities/calendar/types/CalendarEvent.ts:1]: Thank you, I made the corresponding changes! I also pushed a commit that improves calendar events, months and days sorting.

**Comment by bosiraphael** [comment]
LGTM

**Pull request merged**